### PR TITLE
Add cygwin support

### PR DESCRIPTION
**Pull request by marlemion**
Hi, I am successfully using these modifications to get pyinstaller to run a PyQt5 program under windows/cygwin. Decision for cygwin is that I have to use a third party library, which is only POSIX compatible at the moment.

While it is working, I have sometimes signal/slot problems in PyQt5. However, I do not think that they are related to pyinstaller.

I would be happy for a review.

### CODE REVIEW

**Comment by Legorooj** [comment]
Please fix all of the errors [here](https://travis-ci.org/pyinstaller/pyinstaller/jobs/651864390?utm_medium=notification&utm_source=github_status)

File: PyInstaller/depend/bindepend.py [hunk 1]
```diff
@@ -601,6 +608,10 @@ def _getImports_ldd(pth):
                 # http://www.trilithium.com/johan/2005/08/linux-gate/
                 continue
 
+            if is_cygwin and lib.lower().find('/cygdrive/c/windows/system') == 0:
```
Comment (htgoebel) [inline PyInstaller/depend/bindepend.py]: Please change this `if` into a `elif` of the check above:
```
if name[:10] in ('linux-gate', 'linux-vdso'):
   …
elif is_cygwin and …
```

File: PyInstaller/utils/hooks/qt.py [hunk 2]
```diff
@@ -425,8 +427,8 @@ def get_qmake_path(version=''):
     "qt5sql":                   (".QtSql",                 "qtbase",           "sqldrivers"),
     "qt5svg":                   (".QtSvg",                 None,               ),
     "qt5test":                  (".QtTest",                "qtbase",           ),
-    "qt5webkit":                (None,                     None,               ),
-    "qt5webkitwidgets":         (None,                     None,               ),
+    "qt5webkit":                (".QtWebKit",              None,               ),
+    "qt5webkitwidgets":         (".QtWebKitWidgets",       None,               "qt5webkit"),
```
Comment (htgoebel) [inline PyInstaller/utils/hooks/qt.py]: @bjones I'm curious: is this change actually related to cygwin? Or related to some version of Qt?

**Comment by htgoebel** [comment]
@marlemion This PR is stale now for more then half a year. Are you going to continue working on this anytime soon? Otherwise we will close it.

**Comment by bwoodsend** [comment]
Closing due to lack of response from the original author. Will re-open if you
(or anyone else who has this issue) gets back to us by commenting below.
